jwala-webapp/src/main/webapp/resources/css/themes/redmond/theme.css

```diff
@@ -1,5 +1,5 @@
 /*!
```

Comment:
Do we actually use this css? I noticed we deleted a whole bunch of other ones

Reply:
This is used/required by jquery ui css